eva-accession-import-automation/create_fasta_from_assembly_report.sh

```diff
@@ -33,6 +33,9 @@ exit_code=0
 touch ${output_folder}/${species}_custom.fa
 touch ${output_folder}/written_contigs.txt
 
+# remove the FASTA index because it might get outdated (EVA-1581)
+rm ${output_folder}/${species}_custom.fa.fai
```

Comment:
Please use the "-f" switch so that no error is raised in case of a non-existent fai file.